terraform/cp4ba/main.tf

```diff
@@ -52,7 +55,11 @@ data "ibm_container_cluster_config" "cluster_config" {
 
 # --------------- PROVISION DB2  ------------------
 module "install_db2" {
-  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/blob/main/modules/Db2"
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/joel_cloud_pak_terraform_issue220/modules/Db2"
+//  source = "../../../terraform-ibm-cloud-pak/modules/Db2"
+    depends_on = [
+    module.create_cluster
+  ]
 
```

Comment:
You have an `enable` variable listed under db2 variables.  But you don't make it a condition of provisioning the db2 module.  You should probably change the name of that variable to `enable_db2`